tests/templates/test_embeddings/test_displacement_emb.py

```diff
@@ -0,0 +1,271 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the DisplacementEmbedding template.HE
+"""
+import pytest
+import numpy as np
+from pennylane import numpy as pnp
+import pennylane as qml
+
+
+class TestDecomposition:
+    """Tests that the template defines the correct decomposition."""
+
+    @pytest.mark.parametrize("features", [[1, 2, 3], [-1, 1, -1]])
+    def test_expansion(self, features):
+        """Checks the queue for the default settings."""
+
+        op = qml.templates.DisplacementEmbedding(features=features, wires=range(3))
+        tape = op.expand()
+
+        assert len(tape.operations) == len(features)
+        for idx, gate in enumerate(tape.operations):
+            assert gate.name == "Displacement"
+            assert gate.parameters[0] == features[idx]
```

Comment:
I am a bit inconsistent in checking all properties of the gates here because it is a lot of work to get this right...leaving it for future!